CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 
 ### Fixed
+## [0.13.0]
```

Comment:
You can just keep it under "unreleased". When I release a version, I'm doing the change log update etc.